### ISSUE TITLE
pCP8-Quantum_CM4_NoKVM_eMMC

### ISSUE THREAD

**Issue by ChouDoFu** [sam0402/pcp-44.1KHz#2]
Sam,

Hello, thank you so much for your hard work.  I'm an American, but my wife is from 中壢!  I am using a CM4 Compute Module 4 IO Carrier board only for my LMS.  I've found that for me, 44.1 sounds the best.  I've tried all the other builds and this one, overlocked to 2205 with overvoltage is just perfect.  

Yesterday I put my Waveshare PCIe to M.2 in the PCIe port with my NVMe SSD but the pCP8 Quantum would not detect it.  I downloaded the official brand new PCP from "https://docs.picoreplayer.org/downloads/" and it detected the NVMe drive, but I miss the sound from your version.  So much more dense and 3D.

Can you modify the pCP8-Quantum_CM4_NoKVM_eMMC and make a pCP8-Quantum_CM4_NoKVM_eMMC_**PCIe** version that can recognize the NVMe card on the PCIe slot?  I will gladly donate 1000NT!

P.S.  I love 伍佰

Allen

**Comment by sam0402** [comment]
Hi ChouDoFu,
It's great idea to support NVMe. I will make a version for eMMC & PCIe. Please wait for a few days.
I am a 中壢ese too!

**Comment by ChouDoFu** [comment]
You are from 中壢!?  I was a Gloria teacher there from 2005-2006!  If you are from 中壢 then you must know the famous "買芭樂 hu~~~~~~~"  Can't wait, I'm so excited to have this custom software from a 中壢 person, it makes it even more special!

**Comment by ChouDoFu** [comment]
![Bala](https://user-images.githubusercontent.com/101572753/158690201-c92bce6f-0f3e-4131-bea6-6096936b80a6.jpg)

中壢 Night Market.  芭樂 lady.  The Best.

**Comment by sam0402** [comment]
I was born and grow up in 中壢. But I moved to Taipei in 2009.
But my mother is still living in 中壢. So I came older home every couple of weeks.
I went to 中壢 Night Market is so few.
The 芭樂 lady looks not like Taiwanese. She maybe comes from Southeast Asia.

**Comment by ChouDoFu** [comment]
Yes she is definitely 外勞 for sure.  I miss Taiwan so much, I hope I can go back this year.  Taipei is excellent.  I remember the old鼎泰豐 in the East Section before it closed down.  It tastes so much better than the one in the Sogo or the one in Taipei 101.  I think next time I go back I will try to visit some audio stores and purchase some CDs.  Also, I am thinking of getting an X86 so I can try your Arch program too, very grateful that you do this programming for linux.

**Comment by sam0402** [comment]
Hi ChouDofu, 
You can download the new version as below urls.
[https://raw.githubusercontent.com/sam0402/pcp-44.1KHz/master/pCP8-Q176%2B44.img.7z.001](url)
[https://raw.githubusercontent.com/sam0402/pcp-44.1KHz/master/pCP8-Q176%2B44.img.7z.002](url)

**Comment by ChouDoFu** [comment]
I just tried "pCP8-Q176+44" and I can see the IP address on my network for 5 seconds, then it disappears.  I will try the 'Wifi' version now.  I use ethernet connection.  Very excited about these, I'll let you know how it goes and I haven't forgotten about your NT!

Update:  I tried to 'Wifi' version as well.  Same thing happened, the IP address shows up and then disappears and it is no longer on the network.  All the other Quantum versions were fine and I could connect through the Web GUI.  These new versions don't stay on the network so I can't use them yet.  :(  Also want to see if the 2205 Hi Voltage option is there, but I can't access.  Very close!

**Comment by sam0402** [comment]
"pCP8-Q176+44" on my RPi4 is worked find with ethernet. I have no idea what happy on yours.
Is the temperature of CPU is too high? or CM4 can not use "overclock 2205" ? 
Not every RPi4 or CM4 can overclock to 2205. Choose "1764" is great for LMS.

![Q176+44](https://i.imgur.com/tCwdVrm.png)

**Comment by sam0402** [comment]
I find bugs on SD boot, it do't work. I use USB driver works ok.
I will fix it soon.

**Comment by ChouDoFu** [comment]
All of your other versions I flash to EMMC memory on the CM4 board.  I don't use EMMC SD Card, but the EMMC on the CM4.  With your other versions it always works, no problem.  Thank you for looking at it.

**Comment by sam0402** [comment]
I had fix the bugs, please download again.

**Comment by ChouDoFu** [comment]
Sam, everything is working and I am waiting for my wife to finish work so I can listen to music, I will let you know how it sounds compared to M.2 USB.  One question, the old version had 2205 plus 2205 with overvoltage.  Was the overvoltage setting "over_voltage=10"?

**Comment by sam0402** [comment]
The over_vlotage of 2205 is set to "8" in config.txt, so 2205 plus  is "over_voltage=9".

**Comment by ChouDoFu** [comment]
Sam, thank you for all of your work.  I was very curious about the sound with PCIe versus USB.  Honestly, I think the USB sounds better for me!  :)  But I promised you the money, so I bought you 8 coffees!  I see that you are a follower of Jesus?  That is excellent, a Jhong Li brother in Christ and an audiophile, I am very lucky to have your help!

**Comment by sam0402** [comment]
Thanks for a lot of coffee from you. 
The last version of pCP8 is more better than before in sound quality, I am so amazing on it.
Yes, I am a follower of Jesus. My works in audio always got the help from God.

**Comment by ChouDoFu** [comment]
Sam, I saw that the latest build has the ASIX drivers.  I have an Allo USBridge Signature that uses the RPI 3B+ on there.  Right now the best version of piCoreplayer for that is the standard version with some tweaks from Soundcheck (https://soundcheck-audio.blogspot.com/p/the-skit-pcp.html).  I tried loading the latest version of your build and it wouldn't even boot up.

I know many people have the Allo USBridge and would love to have the best player (your pcp version) and I was wondering if you can make a version for all of us with the USBridge.  I know other people on STSD99 have asked, but I wanted to ask as well and I will buy you more coffee for your hard work!

**Comment by sam0402** [comment]
The pCP8-44.1 is loading so heavy for RPi3 that can not boot, even for RPi4 must overclock just working well, thus RPi4 or CM is the lowest requirement.
The CM4_to_CM3_Adapter (https://wiki.geekworm.com/CM4_to_CM3_Adapter) maybe is good solution for USBridge.
I known some people use it for USBridge.

**Comment by ChouDoFu** [comment]
I got lucky and found a CM4 on https://rpilocator.com/ and it will arrive on Monday.  I will try on my USBridge, thank you!

**Comment by eggysu** [comment]
Dear ChouDoFu/Sam0402,

Happy CNY, and i'm from Malaysia and a newbie ^^

I have recently bought Mercury Streamer V2 from Pi2Design. 
[https://www.pi2design.com/mercury-streamer-v2.html](url)

It consisted of CM4 is the 8GB LITE, with PCIe host controller available.

Hence, I bought a WD_Black SN770 M.2 2280 PCIe Gen4 NVMe SSD to flash the pCP8.

However, I have failed to install either sam's pCP8 or official pCP8 from picoreplayer website.

The steps that i followed carefully is based on the website as below:
[https://www.youtube.com/watch?v=NMaqasfN3w0&t=1123s](url)

The failed steps are as below:
1) Flashed sam's all pCP8 program or official pCP8 to SSD card by using Raspberry PI Imager.
2) Plug the SSD to streamer and boot up it but failed.
3) Used Advanced IP Scanner/DHCP Client list from router-can't locate the streamer IP address
4) Search http://pcp.local/ and nothing.
5) Observation= Once boot up, LED_ACT(Green LED, CM4 Activity) was very active flashing as compare of installing official pCP8
6) It seems the streamer was working, but can't access to pcp.local for further action

After failure of testing, proceed with official/sam's pcp8 program installation to streamer via USB flash drive and SD card, no issue and unit running.

Would appreciate if guys cant enlighten flashing and installation via SSD has different steps as compare to youtube above?

**Comment by sam0402** [comment]
Please flash the pcp8 image to SD Card,  not to SSD.
